setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name="aioaseko",
-    version="0.0.2",
+    version="0.0.3",
```

Comment:
Update contains major and breaking changes to the API.

```suggestion
    version="0.1.0",
```